tests/test_defaults.py

```diff
@@ -73,7 +73,7 @@ def test_host_nonexistent_callback(self):
                 "Could not import 'django_hosts.non_existent'. "
                 "Callable does not exist in module",
                 lambda: api_host.callback)
-        except AssertionError:  # Django < 1.8
+        except Exception:  # Django < 1.8
```

Comment:
I've no idea why this is required. I tried printing the type of the exception and it was an instance of `exceptions.AssertionError`.